spec/requests/admin/lead_providers/lead_provider_users_spec.rb

```diff
@@ -61,19 +61,22 @@
     end
   end
 
-  describe "DELETE /admin/suppliers/lead_providers/users/:id/" do
-    it "marks the lead_provider profile as deleted" do
+  describe "DELETE /admin/suppliers/lead_providers/users/:id/", versioning: true do
```

Comment:
What does `versioning: true` do?

Reply:
It enables `PaperTrail` for that block, it's not enabled in specs by default